### PR TITLE
feature/condo/DOMA-1623/warranty ticket type

### PR DESCRIPTION
**Pull request by AleX83Xpert**
New ticket type. `Warranty`.
image
image
image
image

### CODE REVIEW

**Review by akarjakin — APPROVED** [review]

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx [hunk 1]
```diff
@@ -113,9 +114,16 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                         </Tag>
                     )
                 }
+                {
+                    record.isWarranty && (
+                        <Tag color='purple'>
```
Comment (dkoviazin) [inline apps/condo/domains/ticket/hooks/useTableColumns.tsx:119]: Better to move colors to apps/condo/domains/ticket/constants/style.js and give it a name.

File: apps/condo/pages/ticket/[id]/index.tsx [hunk 2]
```diff
@@ -342,6 +344,7 @@ export const TicketPageContent = ({ organization, employee, TicketContent }) =>
                                     <Space direction={'horizontal'} style={{ marginTop: '1.6em ' }}>
                                         {isEmergency && <TicketTag color={'red'}>{EmergencyMessage.toLowerCase()}</TicketTag>}
                                         {isPaid && <TicketTag color={'orange'}>{PaidMessage.toLowerCase()}</TicketTag>}
+                                        {isWarranty && <TicketTag color={'purple'}>{WarrantyMessage.toLowerCase()}</TicketTag>}
```
Comment (dkoviazin) [inline apps/condo/pages/ticket/[id]/index.tsx:347]: Better to move colors to styles

**Review by dkoviazin — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

**Review by sitozzz — APPROVED** [review]
Looks good!

**Review by dkoviazin — CHANGES_REQUESTED** [review]
Please, rename migration file

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/pages/ticket/[id]/index.tsx (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/pages/ticket/[id]/index.tsx:347]: done

File: apps/condo/domains/ticket/hooks/useTableColumns.tsx (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/ticket/hooks/useTableColumns.tsx]: Done

**Comment by AleX83Xpert** [comment]
> Please, rename migration file

Decided to keep filename as is.

**Pull request merged**